tools/crm_mon.c

```diff
@@ -722,16 +722,9 @@ static GOptionEntry display_entries[] = {
 };
 
 static GOptionEntry deprecated_entries[] = {
-    /* @COMPAT At least the following resource agents use --as-xml at time of
-     * writing:
-     * * galera
-     * * pgsql
-     * * rabbitmq-cluster
-     * * redis
-     *
-     * Removing this option must wait until those are updated and we no longer
-     * support building on any platforms that ship the older agents in stock
-     * packages.
+    /* @COMPAT resource-agents <4.15.0 uses --as-xml, so removing this option
```

Comment:
This reads as though `resource-agents` 4.15.0 no longer uses `--as-xml`, which is not the case. The main branch still does.

Reply:
I thought that was fixed in 4.15.0

Reply:
never mind, it did get fixed: it chooses the correct argument based on the pacemaker feature set

Reply:
ah yep, my bad